docs/zh/usage/java-sdk-user-guide.md

```diff
@@ -1187,3 +1187,19 @@ public class SpringIntegrationTest {
   }
 }
 ```
+
+# 七、apollo-client定制
```

Comment:
I think it's better to also update the [English user guide](https://github.com/apolloconfig/apollo/blob/master/docs/en/usage/java-sdk-user-guide.md).

Reply:
Add English version already